src/pages/VenuePage/VenuePage.tsx

```diff
@@ -299,6 +301,8 @@ const VenuePage = () => {
     history.push(`/account/profile?venueId=${venueId}`);
   }
 
+  mixpanel.track("Page load", { venueId: venue.id, template: venue.template });
+
```

Comment:
This will run every single time this component function is run, so a lot. We probably want this in a `useEffect` call that only runs a single time (so empty deps array)

```suggestion
useEffect(() => {
  mixpanel.track("VenuePage loaded", { venueId: venue.id, template: venue.template });
}, []) 
```

This may potentially cause an eslint warning due to not including all of the dependencies.. it depends on your intended usage of this track here as to what makes the most sense, but I think it would be fine if this is called whenever the `venueId` or `template` changes? If so, I would probably do:

```suggestion
const venueId = venue.id
const venueTemplate = venue.template

useEffect(() => {
  mixpanel.track("VenuePage loaded", { venueId, template: venueTemplate });
}, [venueId, venueTemplate]) 
```